docs/docs/20-quickstart.md

```diff
@@ -568,269 +546,42 @@ the previous section.
     version of the `public.ecr.aws/nginx/nginx` container image.
     :::
 
-1. We'll use it later, so save the ID of the `Freight` to an environment
-   variable:
-
-    ```shell
-    export FREIGHT_ALIAS=$(kargo get freight --project kargo-demo --output jsonpath={.alias})
-    ```
-
-1. Now, let's _promote_ the `Freight` into the `test` `Stage`:
-
-    ```shell
-    kargo promote --project kargo-demo --freight-alias $FREIGHT_ALIAS --stage test
-    ```
-
-    Sample output:
-
-    ```shell
-    promotion.kargo.akuity.io/test.01j2wbtrym4r3tktv38qzteh5h.7a6e91f promotion created
-    ```
-
-    Query for `Promotion` resources within our project to see one has been
-    created:
+### Promote to Test
 
-    ```shell
-    kargo get promotions --project kargo-demo
-    ```
+1. To promote `Freight` to the `test` `Stage`, select the existing `Freight` from
+    the Freight Timeline above. Click the three dots, choose <Hlt>Promote</Hlt>,
+    and then select <Hlt>Promote</Hlt> that appeared on the `test` `Stage`.
```

Comment:
This is one of two ways to do it. You can pick a piece of Freight and then the Stage to deploy it to (as you are doing here) or you can pick a Stage and then the Freight you wish to deploy to it.

The latter is more "classic" behavior, with the former having been introduced more recently. To some users, one approach will feel more natural, to others, the other approach.

imho, the "stage first" approach will probably be more intuitively understood by a higher proportion of readers than the "freight first" approach.